### DISCUSSION TITLE
Is there a way to suppress skip messages with reporter = "verbose"?

### DISCUSSION THREAD

**Discussion by psanker** [ropensci/targets#683] [Help] [answered]
I have an "ELT" (ish) pipeline with hundreds of targets, many of which are frequently not invalidated. When I'm working on a couple data models, I'd like to only see the output related to the targets that ran. I have tried using `reporter = "summary"`, but that messes with output from `cat()` that happens in a few packages.

Is there an approach to suppressing these skip messages so I can only focus on the ones that ran?

**Comment by wlandau** [comment, marked as answer]
Enough people have asked at this point that I have just added a couple new reporters (https://github.com/ropensci/targets/commit/ec04c606632771b504a7afb9b7b1bc0228e228db). The new `"verbose_positives"` and `"timestamp_positive"` reporters are like `"verbose"` and `"timestamp"` except without messages for skipped targets.
